File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int height ;
   
   do
    {
        printf("Give a nomber from 0 to 23:");
        height = GetInt();
    }
    while (height < 0 || height > 23);
  
    
  for (int i = 0; i < height; i++)
   {
       int  middle = height + 1 - i - 2;
       for (int j = 0 ; j < middle  ; j++)
       {
             printf (" "); 
       }
          for (int j = middle; j <= height ; j++)
       {
             printf ("#"); 
       }
      printf ("\n");
   }
 return 0;  
}

File: mario.py
#!usr/bin/python3

import sys
import cs50

def main():
	while True:
		print("Height: ", end="")
		height = cs50.get_int()
		if height >= 0 and height <= 23:
			break

	print(height)

	for i in range(height):
		print(" " * (height-i-1), end="")
		print("#" * (i+2))
	return 0

if __name__ == "__main__":
	ret = main()
	exit(ret)